Mathlib/Topology/NhdsSet.lean

```diff
@@ -177,3 +177,65 @@ lemma Continuous.tendsto_nhdsSet_nhds
     Tendsto f (𝓝ˢ s) (𝓝 y) := by
   rw [← nhdsSet_singleton]
   exact h.tendsto_nhdsSet h'
+
+theorem IsOpen.forall_near_mem_of_subset {s t : Set X} (h : IsOpen s) (ht : t ⊆ s) :
+    ∀ᶠ x in 𝓝ˢ t, x ∈ s := by
```

Comment:
[IsOpen.mem_nhdsSet](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/NhdsSet.html#IsOpen.mem_nhdsSet) is the `simp`-normal version of this lemma upgraded to `iff`. Also, we use `eventually_nhdsSet`, not `forall_near` in Mathlib.